FuelGaugeReader/FuelGaugeReader.ino

```diff
@@ -0,0 +1,37 @@
+#define USE_USBCON
+#include <ros.h>
+#include <std_msgs/Int64.h>
+#include <Arduino.h>
+std_msgs::Int64 fuel_level_percent_full;
```

Comment:
I think you may have resolved my comment about moving this to a float. I think you are losing about 3X the resolution by clipping this from a float to an int. 

You said you have a swing from empty to full of ~300 bits which should get you a resolution of about 0.3% but you clip to an int. Plus this doesn't round appropriately. If you calculation of fuel_level_percent_full comes out to 5.99 and you convert to an int it will go to 5. ( I believe)

Reply:
I've been thinking about it that all the calculations are done in doubles, in order to maintain precision during calculation, and then I just output it as an int (I was originally doing int8, only have it like this for testing). If you think the extra decimals from float are useful I can change that, but I don't think It'll actually do anything other than letting it display a decimal place on the big discrete steps

Reply:
Yeah just checked w/ a multimeter, it does have a wide range of possible sensitivities but it's in big discrete steps